katello/hooks/pre/30-upgrade.rb

```diff
@@ -34,9 +34,11 @@ def migrate_foreman
 end
 
 def postgresql_10_upgrade
+  start_postgresql
```

Comment:
Due to line 82 you should be able to count on PG already running. This avoids a duplicated log for the end user (since `Kafo::Helpers.execute` is verbose).

Reply:
I thought about that, but if something errors you can potentially end up in a scenario where this continually fails (as the line after this won't work). I wanted to err on the side of better safe than sorry.

Reply:
Since `systemctl` is quiet, should we call it quietly by passing in `do_say` as false?

Also note that execute returns a boolean whether it succeeded but doesn't actually halt execution. @evgeni noticed that elsewhere as well. We should handle that so it actually stops the upgrade attempt.

Reply:
I'm not sure I get the problem with this logging what it's doing. I'd rather it have that as someone debugging an issue than not.

And yes, planning to address after this fix goes in: https://projects.theforeman.org/issues/28496

Reply:
My concern is that the user will see `systemctl start postgresql` logged twice during an upgrade which may look odd. Not a critical issue and probably worth a larger refactoring to see how we want to deal with output. I hope we'll get to that when we merge clean up the hooks and merge them with the foreman scenario.